zipline/finance/performance/period.py

```diff
@@ -402,11 +429,12 @@ def as_account(self):
                     self.ending_cash / (self.ending_cash + self.ending_value))
         account.day_trades_remaining = \
             getattr(self, 'day_trades_remaining', float('inf'))
-        account.leverage = \
-            getattr(self, 'leverage', self._gross_leverage())
-        account.net_leverage = self._net_leverage()
-        account.net_liquidation = \
-            getattr(self, 'net_liquidation', self._net_liquidation_value)
+        account.leverage = getattr(self, 'leverage',
```

Comment:
In this case we are providing a default of `period_stats.gross_leverage` when a `self.leverage` is not an attribute.

(I'd lean towards checking for a `None` value instead, or some other way of determining whether the default should be used, but not in scope of this change.)